utils/gen_srg_table.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/python3
+
+import argparse
+import collections
+import os
+import xml.etree.ElementTree as ET
+
+import ssg.build_yaml
+import ssg.constants
+import ssg.jinja
+from ssg.constants import PREFIX_TO_NS as NS
+from utils.template_renderer import FlexibleLoader
+
+Srg = collections.namedtuple(
+    "SRG", [
+        "cci", "requirement", "vuln_discussion", "check_content", "fixtext"])
+
+
+def parse_srgs(srgs_filename):
+    srgs = dict()
+    root = ET.parse(srgs_filename)
+    for rule in root.findall(".//xccdf-1.1:Rule", NS):
```

Comment:
`utils/create_srg_export.py` has some code that looks exactly like this. Should we move it to `ssg` and share?

Reply:
great, thanks, done it the latest commit